packages/build/src/error/type.js

```diff
@@ -158,6 +159,13 @@ const TYPES = {
     rawStack: true,
     severity: 'error',
   },
+
+  // Errors related with the telemetry output
+  telemetry: {
```

Comment:
As mentioned in your PR message, we might want to add a test for this new error type (we have a test for almost each of the current error types).